netbox_docker_plugin/templates/netbox_docker_plugin/image.html

```diff
@@ -2,6 +2,83 @@
 
 {% load plugins %}
 
+{% block extra_controls %}
+<button
+  class="btn btn-sm btn-success"
+  onclick="
+    (async () => {
+      this.setAttribute('disabled', 'disabled')
+
+      const notificationData = {}
+
+      try {
+        const resp = await fetch(
+          '/api/plugins/docker/images/{{ object.pk }}/force_pull/',
+          {
+            method: 'POST',
+            headers: {
+              'X-CSRFToken': window.CSRF_TOKEN,
+            },
+          },
+        )
+
+        const data = await resp.json()
+        console.log(data)
```

Comment:
needed ?

Reply:
yes, logs are always good, especially for the day we need to debug :)

Reply:
debug sould exist only on your local env or in the testing phase not on a depoyed version :)